packages/backend/server/src/core/config.ts

```diff
@@ -22,6 +22,21 @@ export function ADD_ENABLED_FEATURES(feature: ServerFeature) {
   ENABLED_FEATURES.add(feature);
 }
 
+@ObjectType()
+export class PasswordLimitsType {
+  @Field()
+  minLength!: number;
+
+  @Field()
+  maxLength!: number;
+}
+
+@ObjectType()
+export class AuthPrivilegeType {
```

Comment:
```suggestion
export class AuthPolicyType {
```

I think to express overall authorization rules or policies, choosing `AuthPolicy` might be more appropriate. Whereas if you need to specifically describe the permissions or privileges of a particular user or entity, then you should use `AuthPrivilege`